engine/src/test/java/org/camunda/bpm/engine/test/api/optimize/GetRunningHistoricTaskInstancesForOptimizeTest.java

```diff
@@ -272,6 +277,26 @@ public void fetchOnlyRunningTasks() {
     assertThat(runningHistoricTaskInstances.get(0).getTaskDefinitionKey(), is("userTask2"));
   }
 
+  @Test
+  public void doNotReturnRunningStandaloneTasks() {
+    try {
+      // given
+      Task task = taskService.newTask("standaloneTaskId");
+      taskService.saveTask(task);
+
+      // when
+      List<HistoricTaskInstance> runningHistoricTaskInstances = optimizeService.getRunningHistoricTaskInstances(null,
+          null, 10);
+
+      // then
+      assertTrue(runningHistoricTaskInstances.isEmpty());
+    } finally {
+      for (Task task : taskService.createTaskQuery().list()) {
+        taskService.deleteTask(task.getId());
+      }
```

Comment:
I added this cleanup here instead of the @after method because this cleanup failed for the other tests.

Reply:
👍 All good.